docs/features/work-log.md

```
@@ -0,0 +1,206 @@
+- 表題は `# 作業ログ YYYY-MM-DD（曜）`（曜日は漢字1文字。日報の表題と同じ形式）とする
+- タイムスタンプは各レコードの `created_at` のローカル時刻を `HH:mm`（24時間・0詰め）で表す
+- 種別ごとの行フォーマット（`{}` は動的値）:
+  - `task_start`: `- {HH:mm} 着手: {タスク名}`。`expected_minutes` があれば末尾に `（予定 {n}分）` を付す
+  - `task_update`: `- {HH:mm} タスク更新: {タスク名}`
+  - `break_start`: `- {HH:mm} 休憩開始`。`expected_minutes` があれば末尾に `（予定 {n}分）` を付す
+  - `break_end`: `- {HH:mm} 休憩終了`
+  - `checkin`: `- {HH:mm} チェックイン`
+  - 決定（`active`）: `- {HH:mm} 決定: {content}`
+  - 決定（`revised`）: `- {HH:mm} 決定（改訂済み）: {content}`
+  - 決定（`withdrawn`）: `- {HH:mm} 決定（撤回）: {content}`
+- `note` の付与規則（種別横断の共通規則）: activity_events の行（`task_start` / `task_update` / `break_start` / `break_end` / `checkin`）は `note` があれば行末（`expected_minutes` による `（予定 {n}分）` があればその後）に ` — {note}` を付す
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)